tests/security/ima/evm_protection_digital_signatures.pm

```diff
@@ -3,16 +3,17 @@
 #
 # Summary: Test EVM protection using digital signatures
 # Note: This case should come after 'evm_protection_hmacs'
-# Maintainer: llzhao <llzhao@suse.com>
-# Tags: poo#53582, poo#92347
+# Maintainer: llzhao <llzhao@suse.com>, rfan1 <richard.fan@suse.com>
+# Tags: poo#53579, poo#100694, poo#102311
 
-use base "opensusebasetest";
+use base 'opensusebasetest';
 use strict;
 use warnings;
 use testapi;
 use utils;
-use bootloader_setup "replace_grub_cmdline_settings";
-use power_action_utils "power_action";
+use bootloader_setup 'replace_grub_cmdline_settings';
+use power_action_utils 'power_action';
+use bootloader_setup 'tianocore_disable_secureboot';
```

Comment:
Looks it can be amended too,
`use bootloader_setup qw(replace_grub_cmdline_settings tianocore_disable_secureboot);`